packages/core/test/types.test.ts

```diff
@@ -1,22 +1,23 @@
-import Bugsnag, { Client, Config } from '..'
```

Comment:
Not sure what was going on here but `@bugsnag/core` never had a default export but this is a confusion from mising cjs exports and esm imports. In cjs it would have been:

```
const { Client, Breadcrumb, Event, Session } = require('@bugsnag/core')` which makes sense against the deleted index file above
```